### ISSUE TITLE
Thread derived class Run method

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#31]
The following program crashes because the thread's ```Run``` method is in a derived class.

```
class Foo {
	function : Main(args : String[]) ~ Nil {
		each(i : 5) {
			f := Fie->New();
			f->Execute(Nil);
		};

		System.IO.Console->ReadString();
	}
}

class Fie from Fum {
	New() {
		Parent();
	}

	method : SayIt() ~ Nil {
		"Hello World!"->PrintLine();
	}
}

class Fum from System.Concurrency.Thread {
	New() {
		Parent();
	}

	method : virtual : SayIt() ~ Nil;

	method : public : Run(param : Base) ~ Nil {
		SayIt();	
	}
}```

**Comment by objeck** [comment]
Fixed in v5.9.3

**Comment by objeck** [comment]
Fixed in v5.9.3
